cypress/e2e/tegb_ddt_tests.cy.js

```diff
@@ -0,0 +1,53 @@
+import { AccountsCreateApi } from "../api/tegb/accounts_create_api";
+import { UserApi } from "../api/tegb/login_api";
+import { LoginPage } from "../page-objects/login_page";
+import { faker } from "@faker-js/faker";
+import accountBalanceData from "../fixtures/account_balance_data.json";
+
+describe("TegB Banking app DDT Tests", () => {
+  let randomEmail;
+  let randomUsername;
+  let randomPassword;
+
+  beforeEach(() => {
+    randomEmail = faker.internet.exampleEmail();
+    randomUsername = faker.internet.userName();
+    randomPassword = faker.internet.password();
+    new LoginPage()
+      .openTegB()
+      .clickRegister()
+      .typeUsername(randomUsername)
+      .typePassword(randomPassword)
+      .typeEmail(randomEmail)
+      .clickSubmit();
+  });
+  accountBalanceData.forEach((accountData) => {
+    it(`DDT test for account balance: ${accountData.startBalance}`, () => {
```

Comment:
Test provádí i nevalidní testy (500 chyba u velkých částek). Tyto testy bych vyjmul z JSON a tady dal poznámku, proč se neexekují.

Reply:
Je možné udělat v JSON disabled element s popisem, proč není kombinace testovaná a pak zde udělat if(), který by toto řešil. Toto ale nebylo vůbec vyžadované, jen posílám inspiraci :-)

Mohlo by to vypadat třeba takto:
**JSON:**
```json
  {
    "row": 4,
    "accountName": "Test_Account_",
    "balance": "1234567890.00",
    "disabled": true,
    "disabledReason": "BUG123 - 500 internal server error, in account creation"
  }
```

**Test:**
``` javascript
import accountsData from "../fixtures/account_balance_data.json";

describe("Disabled exampled", () => {
  accountsData.forEach((accountData) => {
    if (!accountData.disabled) {
      it(`create account no. ${accountData.row + 1}`, () => {
        // test code
      });
    } else {
      it.skip(`${accountData.disabledReason} - DDT Test`, () => {
        // Tady nic nedelame, jenom vypisujeme, ze test je disabled
      });
    }
  });
});
```


A výsledek v Cypress
![image](https://github.com/deni-adam/cypress-bank-app/assets/95618620/d2395c6c-7cb4-4d9b-80e4-1c42aac996c8)